app/pages/project/index.cjsx

```diff
@@ -311,7 +312,7 @@ ProjectPage = React.createClass
       {unless @props.project.launch_approved or @props.project.beta_approved
         <Translate component="p" className="project-disclaimer" content="project.disclaimer" />}
 
-      <PotentialFieldGuide project={@props.project} />
+      <PotentialFieldGuide project={@props.project} onHomePage={onHomePage} />
```

Comment:
Instead of passing down a prop, you can just directly check here and not render. In coffeescript, something like this should work:
```
unless @props.location.pathname is "/projects/#{@props.project.slug}"
  <PotentialFieldGuide />
```